modules/shortcodes/wufoo.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Wufoo shortcode
+ */
+
+function wufoo_shortcode( $atts ) {
+	extract( shortcode_atts( array(
+		'username'   => '',
+		'formhash'   => '',
+		'autoresize' => true,
+		'height'     => '500',
+		'header'     => 'show',
+		'ssl'        => ''
+	), $atts ) );
+
+	//Check username and formhash to ensure they only have alphanumeric characters or underscores, and aren't empty
+	if ( !preg_match( "/^[a-zA-Z0-9_]+$/", $username) || !preg_match( "/^[a-zA-Z0-9_]+$/", $formhash ) ) {
+
+		/**
+		 * Return an error to the users with instructions if one of these params is invalid
+		 * They don't have default values because they are user/form-specific
+		 */
+		return "
+			<div style='border: 20px solid red; border-radius: 40px; padding: 40px; margin: 50px 0 70px;'>
+				<h3>Uh oh!</h3>
+				<p style='margin: 0;'>Something is wrong with your Wufoo shortcode. If you copy and paste it from the <a href='http://wufoo.com/docs/code-manager/'>Wufoo Code Manager</a>, you should be golden.</p>
```

Comment:
Accidently deleted @jeherve's comment, but he's right in that this will need to be translated